sdk/eventhub/Azure.Messaging.EventHubs.Shared/src/Testing/EventHubScope.cs

```diff
@@ -115,9 +188,9 @@ public async ValueTask DisposeAsync()
         ///
         /// <returns>The <see cref="EventHubScope" /> in which the test should be executed.</returns>
         ///
-        public static async Task<EventHubScope> CreateAsync(int partitionCount,
-                                                            IEnumerable<string> consumerGroups,
-                                                            [CallerMemberName] string caller = "")
+        private static async Task<EventHubScope> CreateEventHubAsync(int partitionCount,
```

Comment:
nit:  Can we consider renaming to something like `CreateEventHubAndBuildScope` or `BuildScopeWithNewEventHub`?

Reply:
Good point.

To pair it with ```BuildScopeFromExistingResource```, as we discussed in another thread, do you think would be worth using ```EventHub``` or ```Resource``` in both of them?

Like:

- ```BuildScopeFromExistingResource``` and ```BuildScopeWithNewResource```

Or:

- ```BuildScopeFromExistingEventHub``` and ```BuildScopeWithNewEventHub```

Reply:
You betcha.    I don't have a strong preference between them.   It would probably be helpful to align on `EventHub` just to keep terminology familiar since we're focused there and not specifically on consumer groups or other supporting resources, but I'll trust your judgement there.